commons-numbers-combinatorics/src/main/java/org/apache/commons/numbers/combinatorics/FactorialDouble.java

```diff
@@ -38,7 +38,7 @@
      *
      * @param numValues Number of values of the function to compute.
      * @param cache Cached values.
-     * @throw IllegalArgumentException if {@code n < 0}.
+     * @throw CombinatoricsException if {@code n < 0}.
```

Comment:
`CombinatoricsException` is package private; it should not be exposed in the javadoc. It extends IllegalArgumentException so the javadoc is still valid.

Reply:
Changed